### PR TITLE
Added SchoolCertifyingOfficialValidator

### PR DESCRIPTION
**Pull request by devin-mccurdy**
## Description
Added `SchoolCertifingOfficalValidator` to validate uploaded SCO priority values.

[ZenHub Issue](https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/8947)

## Testing done
Tested locally and QA reviewed

## Screenshots
image

## Acceptance criteria
- [x] A validation message for the data row, "Priority is not a valid value." is displayed when the contents of the "Priority" column do not match one of the following non-case sensitive values:
  - Primary
  - Secondary
- [x] The "School Certifying Official File Data Requirements" informational blue box displays the data requirements for the "Priority" column.

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by zurbergram — APPROVED** [review]

**Review by shawkey-daniel — APPROVED** [review]
LGTM

File: app/validators/school_certifying_official_validator.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class SchoolCertifyingOfficialValidator < ActiveModel::Validator
+  REQUIREMENT_DESCRIPTIONS = ["Valid priority values: #{SchoolCertifyingOfficial::VALID_PRIORITY_VALUES}"].freeze
```
Comment (LindseySaari) [inline app/validators/school_certifying_official_validator.rb:4]: Is REQUIREMENT_DESCRIPTIONS used anywhere?

Comment (zurbergram) [reply app/validators/school_certifying_official_validator.rb:4]: https://github.com/department-of-veterans-affairs/gibct-data-service/blob/master/app/controllers/uploads_controller.rb#L47

Comment (LindseySaari) [reply app/validators/school_certifying_official_validator.rb:4]: Ah, I see it's used in the uploads controller

**Review by LindseySaari — APPROVED** [review]

File: app/validators/school_certifying_official_validator.rb (hunk 1 above)
Comment (dmccurdybah) [reply app/validators/school_certifying_official_validator.rb:4]: Yes, it's used on the upload page for the related model:

image

**Review by delli-gatti-michael — APPROVED** [review]

**Pull request merged**